src/openvslam/tracking_module.cc

```diff
@@ -333,15 +360,32 @@ bool tracking_module::track_current_frame() {
         if (!succeeded) {
             succeeded = frame_tracker_.robust_match_based_track(curr_frm_, last_frm_, ref_keyfrm_);
         }
+    } else if (update_pose_is_requested()) {
+        // Force relocalization by pose
+        curr_frm_.set_cam_pose(request_pose_);
+
+        curr_frm_.compute_bow();
+        const auto candidates = map_db_->get_close_keyframes(request_pose_, update_pose_keyframes_);
+        if (candidates.size() != 0) {
```

Comment:
To match the rest of the code in this project, let's do the following:

```suggestion
        if (!candidates.empty()) {
```